openbb_platform/providers/polygon/openbb_polygon/models/company_news.py

```diff
@@ -19,7 +19,7 @@ class PolygonCompanyNewsQueryParams(CompanyNewsQueryParams):
     Source: https://polygon.io/docs/stocks/get_v2_reference_news
     """
 
-    __alias_dict__ = {"symbols": "ticker"}
+    __alias_dict__ = {"symbols": "tickers"}
```

Comment:
Polygon calls the parameter `ticker`, and it also doesn't support multiple symbols to the News endpoint.

Reply:
Says `tickers` on the docs
<img width="681" alt="image" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/88149434/6fbf6dfc-a181-444d-8b2c-951bea7466a8">

Reply:
Fixed.

Most providers support a string of comma-separated symbols so we are keeping it as `symbols`.